packages/x-data-grid/src/utils/createSelector.ts

```diff
@@ -11,6 +11,10 @@ const reselectCreateSelector = createSelectorCreator({
     maxSize: 1,
     equalityCheck: Object.is,
   },
+  devModeChecks: {
+    inputStabilityCheck: 'once',
+    identityFunctionCheck: 'once',
```

Comment:
Looks like these are default values already: https://github.com/reduxjs/reselect/blob/88e7ffd7d46a1aebf866210075c322a0f424bbe1/src/devModeChecks/setGlobalDevModeChecks.ts#L11-L12
No objections to having them set explicitly though 👍🏻

Reply:
Didn't saw them, all good then.

Reply:
I think it's fine, I would expect that reselect doesn't change their defaults without a major. And at most it makes dev mode slower, so not an issue for our users.